src/Writer/Renderer/AbstractRenderer.php

```diff
@@ -167,8 +167,10 @@ public function getExceptions()
      * on their appropriateness for the current type, e.g. renderers.
      *
      * @param string $type
+     *
+     * @return void
      */
-    public function setType($type)
+    public function setType($type): void
```

Comment:
Please do not add php typehints as this will is a BC break unless the class is final.